testing/serial_procedure_test.py

```diff
@@ -0,0 +1,51 @@
+#
+# serial_procedure_test.py
+# datacratic, 2015
```

Comment:
It's the original creation of the original file. I'm unsure whether I should bump it or not.